bin/run-tests.server.lua

```diff
@@ -12,14 +12,15 @@ Roact.setGlobalConfig({
 	["elementTracing"] = true,
 	["propValidation"] = true,
 })
-local results = TestEZ.TestBootstrap:run(ReplicatedStorage.Roact, TestEZ.Reporters.TextReporter)
+local results = TestEZ.TestBootstrap:run(
+	{ ReplicatedStorage.Roact },
+	TestEZ.Reporters.TextReporter
+)
 
-local statusCode = results.failureCount == 0 and 0 or 1
+local statusCode = (results.failureCount == 0 and #results.errors == 0) and 0 or 1
 
 if __LEMUR__ then
-	if results.failureCount > 0 then
-		os.exit(statusCode)
-	end
+	os.exit(statusCode)
 elseif isRobloxCli then
-	ProcessService:Exit(statusCode)
+	ProcessService:ExitAsync(statusCode)
```

Comment:
What's the reason for this change? Just for my own learning.

Reply:
At some point the API just changed 😅 Blocking functions are usually marked with an `Async` suffix that's why it got corrected